pkg/interop/runtime/runtime.go

```diff
@@ -29,7 +29,7 @@ func Log(message string) {}
 // part of contract's API to external systems, these events can be monitored
 // from outside and act upon accordingly. This function uses
 // `System.Runtime.Notify` syscall.
-func Notify(name string, arg ...interface{}) {}
+func Notify(name string, args []interface{}) {}
```

Comment:
But why?

Reply:
Just removed this commit, our notifications work fine on *up-to-date* master.